webdriver-spec.html

```diff
@@ -2731,6 +2731,16 @@
 
  <li><p>Set the <a>webdriver-active flag</a> to true.</p>
 
+ <li><p>Set the <a>current top-level browsing context</a>
+  for <var>session</var> in an implementation-specific way.
```

Comment:
I've mirrored the language from [Section 8](https://w3c.github.io/webdriver/webdriver-spec.html#sessions) that discusses the same thing.